lib/cylc/cylc-review/template/suite-base.html

```diff
@@ -0,0 +1,169 @@
+</head>
+<body>
+
+<!-- Static navbar -->
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed"
+      data-toggle="collapse" data-target="#navbar" aria-expanded="false"
+      aria-controls="navbar">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="{{script}}/">
+        {% if logo -%}
+        <img {{logo}} />
+        {% endif -%}
+        {{title}} <small>@ {{host}}</small>
+      </a>
+    </div>
+    <div id="navbar" class="navbar-collapse collapse">
+      <ul class="nav navbar-nav">
+
+        <li><a href="{{script}}/suites/{{user}}">{{user}}</a></li>
+        <li><a><strong>{{suite}}</strong></a></li>
+        {% for method, name, icon in [
+            ("cycles", "cycles list", "glyphicon-tasks"),
+            ("taskjobs", "task jobs list", "glyphicon-th-list"),
+            ("broadcast_states", "broadcasts list", "glyphicon-bullhorn")] %}
+        {% if name == self.title_name() %}
+        <li class="active">
+          <a>
+            <i class="glyphicon {{icon}}" title="{{name}}"></i>
+            {{name}}
+          </a>
+        </li>
+        {% else %}
+        <li>
+          <a href="{{script}}/{{method}}/{{user}}?&amp;suite={{suite|replace("/", "%2F")}}">
+            <i class="glyphicon {{icon}}" title="{{name}}"></i>
+            {{name}}
+          </a>
+        </li>
+        {% endif %}
+        {% endfor %}
+        {% if files -%}
+        {% for scheme, scheme_files in files|dictsort -%}
+        {% if scheme_files -%}
+        <li class="dropdown">
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+            {{scheme}} files <b class="caret"></b>
+          </a>
+          <ul class="dropdown-menu">
+        {% for key, log in scheme_files|dictsort -%}
+        {% if log.paths and log.paths|length > 1 -%}
+            <li class="dropdown-submenu">
+              <a href="#">{{log.path}}</a>
+              <ul class="dropdown-menu">
+        {% for log_path in log.paths -%}
+                <li>
+                  <a href="{{script}}/view/{{user}}?&amp;suite={{suite|replace("/", "%2F")}}&amp;path={{log_path}}&amp;no_fuzzy_time={{no_fuzzy_time}}"
+                  title="{{log.size}} bytes">{{log_path}}</a>
+                </li>
+        {% endfor -%}
+              </ul>
+            </li>
+        {% else -%}
+            <li>
+              <a href="{{script}}/view/{{user}}?&amp;suite={{suite|replace("/", "%2F")}}&amp;path={{log.path}}&amp;no_fuzzy_time={{no_fuzzy_time}}"
+              title="{{log.size}} bytes">{{key}}</a></li>
+        {% endif -%}
+        {% endfor -%}
+          </ul>
+        </li>
+        {% endif -%}
+        {% endfor -%}
+        {% endif -%}
+        {% if info -%}
```

Comment:
This is an [issue inherited](https://github.com/metomi/rose/blob/master/lib/html/template/rose-bush/suite-base.html) from the original Rose Bush; I have scarcely touched the web-based files. I would like to merge the utility, adapted as necessary for migration but otherwise largely 'as-is', before we address the various shortcomings of Rose Bush. Otherwise this PR could be indefinitely extended outside its domain & reviewing will become ever-more difficult.

Reply:
I am changing this particular minor issue since you have pointed it out, however.

Reply:
Perhaps not, I will investigate.

Reply:
It is not relevant to Cylc so I will just remove it.

Whilst amending this I also noticed a bug whereby Rose ``.version`` files were recognised twice, & have also fixed this in my response commit.